lib/hipchat/user.rb

```diff
@@ -100,5 +99,36 @@ def delete(params = {})
       true
     end
 
+    #
+    # Update status.
+    # API: https://www.hipchat.com/docs/apiv2/method/update_user
+    # Request body
+    # name - User's full name.  Valid length range: 1-50
+    # status - string may be null
+    # show - string - the status to show for the user. Available options 'away', 'chat', 'dnd', 'xa'
+    # mention_name - User's @mention name without the @
+    # email - User's email
+    def status(message, options = {})
```

Comment:
What do you think about calling this `User#update` since it updates the user's name, email, and mention name in addition to the two presence/status fields?